src/app/containers/Ad/Amp/__snapshots__/index.test.jsx.snap

```diff
@@ -81,22 +81,22 @@ exports[`AMP Ads Snapshots should correctly render an AMP leaderboard ad 1`] = `
 }
 
 <div>
-  <section
-    aria-hidden="true"
-    aria-label="Publicités"
+  <div
+    amp-access="toggles.ads.enabled"
+    amp-access-hide="true"
     class="emotion-0 emotion-1"
-    data-e2e="advertisement"
-    role="region"
   >
-    <div
+    <section
+      aria-hidden="true"
+      aria-label="Publicités"
```

Comment:
Is this just for one language?

Reply:
Thats just for the service used in the snapshot test. We don't run snapshots for every service.

Reply:
Cool, thanks Chris